### PR TITLE
Sort Command

### PR DESCRIPTION
**Pull request by Livian1107**
create a new command to sort the persons in addressbook with their names in alphabetical order

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/SortCommand.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Sorts all persons in the address book in alphabetical order.
```
Comment (AdityaA1998) [inline src/main/java/seedu/address/logic/commands/SortCommand.java:6]: Should this be view window incase only a selected results are shown?

**Review by AdityaA1998 — APPROVED** [review]
Overall, its good and doesn't need nay major changes :)

File: src/test/java/seedu/address/logic/commands/SortCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ */
+public class SortCommandTest {
+
+    private Model model;
+    private Model expectedModel;
+    private SortCommand sortCommand;
+
+    @Before
+    public void setUp() {
+        model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
```
Comment (EdwardKSG) [inline src/test/java/seedu/address/logic/commands/SortCommandTest.java:26]: the model is already sorted. should take an UNSORTED one as the addressbook to be sorted, then sort it, and compare to an originally sorted addressbook with exactly the same people.

**Review by EdwardKSG — APPROVED** [review]
overall good, but i suggest you to update the test if i didn't misunderstand your code.

**Pull request merged**